target/linux/ath79/dts/qca953x.dtsi

```diff
@@ -86,7 +86,7 @@
 			pinmux: pinmux@1804002c {
 				compatible = "pinctrl-single";
 
-				reg = <0x1804002c 0x48>;
+				reg = <0x1804002c 0x44>;
```

Comment:
This probably needs to go into separate commit with proper explanation.

Reply:
Sorry, it didn't mean to go in the commit and was later reverted in a following push. I was trying different values, out of desperation.